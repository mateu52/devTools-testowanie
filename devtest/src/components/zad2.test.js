import React from "react";
import Zadtwo from "./zad2";
import renderer from 'react-test-renderer';

it('renders ok', () => {
    const tree = renderer
        .create(<Zadtwo />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})