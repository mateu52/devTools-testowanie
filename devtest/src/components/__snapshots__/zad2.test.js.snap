// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ok 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    +
  </button>
  <button
    onClick={[Function]}
  >
    -
  </button>
  <form
    onSubmit={[Function]}
  >
    <input
      name="number"
      onChange={[Function]}
      placeholder="number"
      type="number"
    />
    <button
      type="submit"
      value="Submit"
    >
      Zmie≈Ñ
    </button>
  </form>
  <h4>
    stan licznika: 
    0
  </h4>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
